miqsel/miqsel.py

```diff
@@ -132,6 +132,14 @@ def get_container(connection):
         return None
 
 
+@connection
+def get_image(connection):
```

Comment:
rather than this...
I would check for image in `start` command. As its not going to `reuse` second time.

so better to check like image is available in `client.images.list()` if not then echo messages on screen.